src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.cmd

```diff
@@ -31,21 +31,45 @@ ROBOCOPY "%BUILDTOOLS_PACKAGE_DIR%\." "%TOOLRUNTIME_DIR%" /E
 set TOOLRUNTIME_PROJECTJSON=%BUILDTOOLS_PACKAGE_DIR%\tool-runtime\project.json
 @echo on
 call "%DOTNET_CMD%" restore "%TOOLRUNTIME_PROJECTJSON%" --source https://dotnet.myget.org/F/dotnet-core/api/v3/index.json --source https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json --source https://api.nuget.org/v3/index.json
+set RESTORE_ERROR_LEVEL=%ERRORLEVEL%
+@echo off
+if not [%RESTORE_ERROR_LEVEL%]==[0] (
+	echo ERROR: An error occured on the restore.
```

Comment:
Since this is the inner init-tools, this error statement will already be *in the log* so that's why I thought that the user can go and find in the log what happened. In the outer init-tools (the one that lives in each repo) I will catch this exit code, and then redirect people to look in the log for more details. If you still think that I should add more verbosity I can certainly do that.